installation.rst

```diff
@@ -1052,3 +1066,43 @@ To delete the lima VM:
 
    $ limactl delete singularity-ce
 
+{Singularity} Docker Image
+==========================
+
+It is also possible to run {Singularity} inside Docker, or another compatible
+OCI container runtime. This may be convenient if you have Docker Desktop, or a
+similar solution, already installed on your PC or Mac.
+
+Docker containers for {Singularity} are maintained at
+https://quay.io/repository/singularity/singularity. 
+
+.. note::
+
+  These containers are maintained by a third party. They are not part of the
+  {Singularity} project, or reviewed by Sylabs.
+
+An example of a suitable ``compose.yaml`` file to start up {Singularity} in a
+Docker container is given below. Note that privileged operation is needed to
+successfully run {Singularity} nested inside of Docker. Change the version
+number on the ``image:`` line to your preferred release.
+
+.. code::
+
+   services:
+     singularity:
+       image: quay.io/singularity/singularity:v3.11.4-slim
+       stdin_open: true
+       tty: true
+       privileged: true
+       volumes:
+         - .:/root
+       entrypoint: ["/bin/sh"]
+
+Singularity in Docker can have various disadvantages, but basic
+container operations will work. Currently, the intended use case is
+continuous integration, meaning that you should be able to build a
```

Comment:
Is [this](https://sylabs.io/2023/08/using-singularity-container-services-registry-for-ci-cd-workflows/) relevant, and is so, should we link to it here?